src/fr/devsylone/fallenkingdom/display/content/ConstantContent.java

```diff
@@ -55,16 +63,48 @@ public boolean containsAny(@NotNull PlaceHolder... placeHolders) {
         return false;
     }
 
+    /**
+     * Vérifie si ce contenu contient des placeholders PAPI
+     * @return true si des placeholders PAPI sont présents
+     */
+    public boolean containsPapiPlaceholders() {
+        return this.hasPapiPlaceholders;
+    }
+
     @Override
     public @NotNull String format(@NotNull Player player, @NotNull FkPlayer fkPlayer, @NotNull PlaceHolder... placeHolders) {
         if (fkPlayer.useFormattedText()) {
             String replaced = this.value;
+
+            // Traitement des placeholders internes
             for (PlaceHolder placeHolder : this.placeHolders) {
                 replaced = placeHolder.replaceMultiple(replaced, player);
             }
+
+            // Traitement des placeholders PAPI si disponible
+            if (this.hasPapiPlaceholders) {
+                try {
+                    replaced = PlaceholderAPI.setPlaceholders(player, replaced);
+                } catch (Exception e) {
+                    // En cas d'erreur PAPI, on continue sans traiter les placeholders PAPI
+                    Bukkit.getLogger().warning("[FallenKingdom] Erreur lors du traitement des placeholders PAPI: " + e.getMessage());
+                }
+            }
+
             return replaced;
         } else {
-            return ChatUtils.translateColorCodeToAmpersand(this.value);
+            String result = ChatUtils.translateColorCodeToAmpersand(this.value);
+
+            // Même pour le texte non formaté, on traite les placeholders PAPI
+            if (this.hasPapiPlaceholders) {
+                try {
+                    result = PlaceholderAPI.setPlaceholders(player, result);
+                } catch (Exception e) {
+                    Bukkit.getLogger().warning("[FallenKingdom] Erreur lors du traitement des placeholders PAPI: " + e.getMessage());
+                }
+            }
+
+            return result;
```

Comment:
```suggestion
            return ChatUtils.translateColorCodeToAmpersand(this.value);
```
Le mode "non formaté" est un mode de debug, j'aurais tendance à dire qu'on s'attend à ce que tout ne soit pas formaté dans ce mode.